addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -377,6 +377,11 @@ def _process_step_forward_operator(self, discuss_channel, users=None):
                 # first post the message of the step (if we have one)
                 posted_message = discuss_channel._chatbot_post_message(self.chatbot_script_id, plaintext2html(self.message))
 
+            if self.step_type == 'forward_operator':
+                # temporay bypass
+                discuss_channel.sudo().write({
+                    'operator_expertise_ids': [(6, 0, human_operator.sudo().res_users_settings_ids.livechat_expertise_ids.ids)]
+                })
```

Comment:
We should be taking the expertise from the forward chatbot step directly (the expertise requested by the user), not the expertise from the agent (which can be different depending on availability).

This should be written on the member history (through create_member_params), not on the channel.